src/components/ChatLog.jsx

```diff
@@ -0,0 +1,46 @@
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+
+
+const ChatLog =(props) => {
+  const messageComponents = props.entries.map((message) => {
+    return (
+      <li key={message.id}>
+        <ChatEntry
+          id={message.id}
+          sender={message.sender}
+          body={message.body}
+          timeStamp={message.timeStamp}
+          liked={message.liked}
+          onLikedToggle={props.onLikedToggle}
+          likesCount={props.likesCount}
+          setLikesCount={props.setLikesCount}
+          calculateLikes={props.calculateLikes}/>
```

Comment:
Since the names of the keys on `entry` are the same as the names your are setting on `ChatEntry` attributes/you are using all of the values in `entry` you could do something something like this to save you a few keystrokes: 

```javascript
  const chatComponents = entries.map((entry) => {
    return(
      <ChatEntry
        {...entry}
        onLikeToggle={onLikeBtnToggle}
        key={entry.id}
      />
    );
  });
```
Just keep in mind that this is not as explicit as what you have above.